va/va.h

```diff
@@ -1401,6 +1408,21 @@ typedef union _VAConfigAttribValContextPriority {
     uint32_t value;
 } VAConfigAttribValContextPriority;
 
+/** brief Attribute value VAConfigAttribEncPerMBControl */
+typedef union VAConfigAttribValEncPerMBControl {
+    struct {
+        /** \brief whether to support dela qp per block */
+        uint32_t delta_qp_support         : 1;
+        /** \brief supported size of delta qp block */
+        uint32_t log2_delta_qp_block_size : 4;
+        /** \brief size of delta qp per block in bytes*/
+        uint32_t delta_qp_per_block_size  : 3;
```

Comment:
How about renaming this member to delta_qp_size_in_bytes ? If so, a VA user needn't go through the comment in va.h and know what this member is used for

Reply:
It is a better name.  I'll rename to it.